src/OrchardCore.Modules/OrchardCore.Contents/Scripting/ContentMethodsProvider.cs

```diff
@@ -51,19 +48,26 @@ public ContentMethodsProvider()
                 Method = serviceProvider => (Action<ContentItem, object>)((contentItem, properties) =>
                 {
                     var contentManager = serviceProvider.GetRequiredService<IContentManager>();
-                    var props = JObject.FromObject(properties);
-                    var content = (JObject)contentItem.Content;
-
-                    content.Merge(props, new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace });
-                    contentItem.Apply(props.ToObject<ContentItem>());
+                    contentItem.Merge(properties, new JsonMergeSettings { MergeArrayHandling = MergeArrayHandling.Replace });
```

Comment:
Why are  you no longer calling .Apply() here ?

Reply:
@jptissot  I introduced a ContentItem extension method, .Merge which merges the Json and clears the elements of the ContentItem.